app/feature_flags_sync.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-package config
+package app
```

Comment:
Would it make sense to have this as a sub-package under app? 0/5.

Reply:
Sure, that works as well.